packages/code-du-travail-frontend/src/outils/IndemniteLicenciement/ccn/1486/steps/AncienneteCE.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import styled from "styled-components";
+import { Field } from "react-final-form";
+import { theme } from "@cdt/ui";
+
+import {
+  InlineError,
+  Input,
+  Label,
+  SectionTitle
+} from "../../../../common/stepStyles";
+import { isNumber } from "../../../../common/validators";
+
+export const ANCIENNETE_CE_KEY = "brancheAncienneteCE";
```

Comment:
using key name is a good idea but, since we can't use in getIndemnite function, I not sure it worth it since it's only used in the step

Reply:
removed !